modules/client/core/app/config.ts

```diff
@@ -59,6 +58,8 @@ const startAngularApp = () => {
         };
       }
     });
+
+    window.state = $state
```

Comment:
make the state service global for now bc we need to change angular state from react state